src/Bicep.LangServer/Configuration/ConfigurationHelper.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using Bicep.Core;
+using Bicep.Core.Configuration;
+using OmniSharp.Extensions.LanguageServer.Protocol;
+
+namespace Bicep.LanguageServer.Configuration
+{
+    public class ConfigurationHelper
+    {
+        public static bool IsBicepConfigFile(DocumentUri documentUri)
+        {
+            try
+            {
+                return string.Equals(Path.GetFileName(documentUri.Path), LanguageConstants.BicepConfigurationFileName, StringComparison.OrdinalIgnoreCase);
+            }
+            catch (Exception e)
+            {
+                Trace.WriteLine($"Encountered issue while trying to get the file name: {e.Message}");
+                return false;
+            }
+        }
+
+        public static bool TryGetConfigurarion(IConfigurationManager configurationManager, DocumentUri documentUri, out RootConfiguration? rootConfiguration)
```

Comment:
Change "out RootConfiguration? rootConfiguration" to:
"[NotNullWhen(false)] out RootConfiguration? rootConfiguration"

Then you won't have to check that rootConfiguration is null when calling the function